main.tf

```diff
@@ -1,3 +1,132 @@
-/********************************************************************
-This file is used to implement the ROOT module.
-*********************************************************************/
+###########################################################
+# This file creates an event notificaiton resource instance
+###########################################################
+locals {
+  # tflint-ignore: terraform_unused_declarations
+  validate_kms_plan = var.kms_encryption_enabled && var.plan != "standard" ? tobool("kms encryption is only supported for standard plan") : true
+  # tflint-ignore: terraform_unused_declarations
+  validate_kms_values = !var.kms_encryption_enabled && (var.kms_key_crn != null || var.root_key_id != null) ? tobool("When passing values for var.kms_key_crn or/and var.root_key_id, you must set var.kms_encryption_enabled to true. Otherwise unset them to use default encryption") : true
```

Comment:
This now needs to be updated since we are removing `kms_key_crn` variable

Reply:
Done.